src/components/grid-view/GridView.scss

```diff
@@ -2,32 +2,41 @@
 @import '../../styles/variables';
 
 .bdl-GridView {
-    // margin: 20px 0 0;
     padding: 0 5px;
 
     .bdl-GridView-row {
         display: flex;
-        justify-content: space-between;
+        justify-content: flex-start;
     }
 
-    &.bdl-GridView--columns-1 .bdl-GridViewSlot {
+    .bdl-GridView-slot {
+        border: 1px solid $bdl-gray-10;
+        margin: 10px 0 0 0;
+        overflow: hidden;
+        padding: 15px;
+        white-space: normal;
+    }
+
+    &.bdl-GridView--columns-1 .bdl-GridView-slot {
         $columns1prct: 100%;
 
-        flex-basis: $columns1prct;
+        flex: 0 1 $columns1prct;
         max-width: $columns1prct;
     }
 
-    &.bdl-GridView--columns-3 .bdl-GridViewSlot {
-        $columns3prct: calc(33.3333% - 6.6666px); /* 100%/3 - 20px/3 */
+    &.bdl-GridView--columns-3 .bdl-GridView-slot {
+        $columns3prct: calc(33.3333% - 10px); /* 100%/3 - 10px */
 
-        flex-basis: $columns3prct;
+        flex: 0 1 $columns3prct;
+        margin-right: 10px;
```

Comment:
I had to include this as a separate rule, not inside `.bdl-GridView`, which may or may not have been what you intended.  If I included inside the `bdl-GridView` like this:
<pre>
.bdl-GridView {
    ...
    .bdl-GridView-slot {
        ...
        & + & {
            margin-left: 10px;
        } 
    }
}
</pre>
The rule would be translated to `.bdl-GridView .bdl-GridView-slot + .bdl-GridView .bdl-GridView-slot`.  Since the css is read from right to left, this would not properly select the GridViewSlots as desired.